cdap-data-fabric/src/test/java/io/cdap/cdap/data2/metadata/system/DatasetSystemMetadataProviderTest.java

```diff
@@ -29,20 +29,6 @@ public class DatasetSystemMetadataProviderTest {
 
   @Test
   public void testFilesetSchema() {
-    DatasetProperties filesetAvroTableProps =
```

Comment:
Please confirm if this needs to be removed.

Reply:
Here, the issue was that setTableProperty() method was removed by me as it was using `PROPERTY_EXPLORE_TABLE_PROPERTY_PREFIX` which was related to explore. Also, in `DatasetSystemMetadataProvider.java`, I had deleted this block of code:
` else if (datasetProperties.containsKey(getExplorePropName(FILESET_AVRO_SCHEMA_PROPERTY))) {
      // Fileset with avro schema (CDAP-5322)
      schemaStr = datasetProperties.get(getExplorePropName(FILESET_AVRO_SCHEMA_PROPERTY));`
 as it was using a method from `ExploreProperties.java`. This resulted in test failures in `DatasetSystemMetadataProviderTest.java`. Now, I have fixed the files so that the tests will pass and no method related to explore will be used. Please review those changes once I commit them. Marking this as resolved.